api.js

```diff
@@ -211,10 +245,24 @@ exports.init = function (sbot, config) {
   }
 
   return {
-    getRoot,
-    findOrCreate,
-    findAndTombstone,
-    findById,
     branchStream,
+    findOrCreate: v1FindOrCreate,
+
+    advanced: {
+      getRoot,
+      findOrCreate,
+      findAndTombstone,
+      findById,
+
+      // TODO add API for whether to encrypt the announce message for a subfeed
+      // for sympathetic replication?
```

Comment:
scope out the API change, make sure this API is flexible-foundation, then do that in different PR

Reply:
https://github.com/ssbc/ssb-meta-feeds/issues/78